src/transforms/glossary.ts

```diff
@@ -27,11 +55,35 @@ export function addGlossaryDefinition(
 
         if (at == null) return
 
+        const fragment: Descendant[] = Editor.fragment(editor, at)
+        const nodeToInsert: Node[] = []
+        const getToBottom = (descendant: Descendant[]) => {
+            if (Array.isArray(descendant)) {
+                descendant.some((elem: Descendant) => {
+                    if (VALID_DEFINITION_TERM_ELEMENTS.some(v => v(elem))) {
+                        if (Array.isArray(elem.children)) {
+                            JSON.parse(JSON.stringify(elem.children)).forEach((elem: Element) => {
```

Comment:
What is this conversion for?

```suggestion
                            for (const elem of elem.children) {
```

Reply:
If I do as you suggest, I get an error:
`TypeError: Cannot assign to read only property 'id' of object`

Reply:
Was that error by any chance in [src/id.ts:36](https://github.com/openstax-poland/cnx-designer/blob/master/src/id.ts#L36)?

Reply:
I am not quite sure about that, there is the link to error message:
https://docs.google.com/document/d/1YLaUgrjJqzpPbS9PFvqLzA2h6wLpv6KEOUIxQH8p1q4/edit?usp=sharing

Reply:
Unfortunately this tells me nothing. To debug this issue and see where it actually occurs open debug tools, go to debugger (sources in chrome), and select “pause on exceptions” (you may also need to select “pause on caught exceptions”), and then trigger this issue again. The debugger will pause, and you'll be able to check exactly where it is happening.